config/game_options.txt

```diff
@@ -223,12 +121,6 @@ PROTECT_ROLES_FROM_ANTAGONIST
 ## If late-joining players have a chance to become a traitor/changeling
 ALLOW_LATEJOIN_ANTAGONISTS
 
-## The maximum amount of extra tickets a user may use from their ticket bank in addition to the default tickets
-MAX_TICKETS_PER_ROLL 100
-
-## Uncomment to allow players to see the set odds of different rounds in secret/random in the get server revision screen. This will NOT tell the current roundtype.
-#SHOW_GAME_TYPE_ODDS
```

Comment:
Keep this config, get @Mothblocks to support it for dynamic.

Reply:
game type odds or antag tickets?

Reply:
It's not feasible, unless you want something as weak as "here is the weight", which is not going to be very useful.

The odds of a ruleset being picked in Dynamic is influenced by:
- The threat level, which is determined on a bell curve
- ...as well as the budget split, which is also on a bell curve
- ...but not just for cost, because if there's not enough threat a lot of rulesets won't attempt to roll, regardless of cost.
- The number of players in the round
- ...but if it's a midround, then the number of players at that time
- ...adjusted by the number of alive antags
- ...and the number of alive players
- ...and adjusted by random events

Reply:
well, if there are no further objections I am marking it ready for review